### PR TITLE
Remove panics when TLS cert/key reading fails

### PR DESCRIPTION
**Pull request by hannesdejager**

### CODE REVIEW

File: src/server/tls.rs [hunk 1]
```diff
@@ -56,26 +56,26 @@ impl TryFrom<FtpsConfig> for tokio_rustls::TlsAcceptor {
     }
 }
 
-pub fn new_config<P: AsRef<Path>>(certs_file: P, key_file: P) -> Arc<rustls::ServerConfig> {
-    let certs: Vec<Certificate> = load_certs(certs_file);
-    let privkey: PrivateKey = load_private_key(key_file);
+pub fn new_config<P: AsRef<Path>>(certs_file: P, key_file: P) -> std::io::Result<Arc<rustls::ServerConfig>> {
```
Comment (Nonius) [inline src/server/tls.rs:59]: suggestion(non-blocking): I prefer `use rustls::ServerConfig`, and I always do them, so looks like we have to decide if we want it consistent and if so, then in what style? Happens multiple places. As I learned it just now `std::io::Result` is better not to `use ...`. https://doc.rust-lang.org/std/io/type.Result.html so keep that one.

File: src/server/tls.rs [hunk 2]
```diff
@@ -56,26 +56,26 @@ impl TryFrom<FtpsConfig> for tokio_rustls::TlsAcceptor {
...
-    let certfile: File = File::open(filename).expect("cannot open certificate file");
+fn load_certs<P: AsRef<Path>>(filename: P) -> std::io::Result<Vec<rustls::Certificate>> {
+    let certfile: File = File::open(filename)?;
     let mut reader: BufReader<File> = BufReader::new(certfile);
-    rustls::internal::pemfile::certs(&mut reader).unwrap()
+    Ok(rustls::internal::pemfile::certs(&mut reader).unwrap())
 }
 
-fn load_private_key<P: AsRef<Path>>(filename: P) -> rustls::PrivateKey {
+fn load_private_key<P: AsRef<Path>>(filename: P) -> std::io::Result<rustls::PrivateKey> {
```
Comment (Nonius) [inline src/server/tls.rs:76]: Suggestion(non-blocking): see above `use rustls::PrivateKey`

**Review by Nonius — APPROVED** [review]

File: src/server/tls.rs (hunk 1 above)
Comment (hannesdejager) [reply src/server/tls.rs]: I agree and fixed it. Kept `std::io::Result`

File: src/server/tls.rs (hunk 2 above)
Comment (hannesdejager) [reply src/server/tls.rs]: I agree and fixed it. Kept `std::io::Result`

**Pull request merged**